string actionName
geometry_msgs/PoseStamped[] endEffectorInfo
